ci/test-pytest.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash -ex
 
 cd /vagrant
-exec pipenv run pytest
+
+echo "MINIO_ADDRESS=10.66.60.1:9000
+MINIO_ACCESS_KEY=1234
+MINIO_SECRET_KEY=123456789
+VMCK_API_URL=http://10.66.60.1:10000/v0/
+DEBUG=True
+HOSTNAME=*
+ACS_INTERFACE_ADDRESS=http://10.66.60.1:8000/
+MANAGER_TAG=vagrant-pytest" > .env
+
+pipenv run pytest --liveserver 10.66.60.1:8000
+
+rm .env
```

Comment:
Q: ```rm -rf```?

Reply:
Actually we might not even need this. It's only run on the CI so we don't care about clean-up.